libs/ach-positive-pay/src/lib/models/ach-rule.ts

```diff
@@ -0,0 +1,6 @@
+export interface AchRule {
+  arrangement: string;
+  companyId: string;
+  companyName: string;
+  paymentTypes: string;
+}
```

Comment:
Add a new line at the bottom of the file

Reply:
fixed